android/src/main/java/org/tiqr/authenticator/identity/IdentityCursorAdapter.java

```diff
@@ -39,4 +40,29 @@ public final void bindView(final View v, final Context context, final Cursor cur
 
         super.bindView(v, context, cursor);
     }
+
+    private class DownloadImageTask extends AsyncTask<String, Void, Bitmap> {
+        ImageView bmImage;
+        public DownloadImageTask(ImageView bmImage) {
+            this.bmImage = bmImage;
+        }
+
+        protected Bitmap doInBackground(String... urls) {
+            String urldisplay = urls[0];
+            Bitmap bmp = null;
+            try {
+                InputStream in = new java.net.URL(urldisplay).openStream();
+                bmp = BitmapFactory.decodeStream(in);
+            } catch (Exception e) {
+                Log.e("Error", e.getMessage());
+                e.printStackTrace();
```

Comment:
You should use `Log.e(DownloadImageTask.class.getName(), "Error fetching bitmap", e);`
This will print your custom message, a better class name, and the exception as well.